addons/product/views/product_views.xml

```diff
@@ -646,22 +646,23 @@ action = {
                         <div class="o_product_catalog_cancel_global_click">
                             <a role="menuitem" type="open" class="dropdown-item border-top-0">Edit</a>
                         </div>
+                        <div class="container p-0">
+                            <field name="is_favorite" widget="boolean_favorite" class="col-12 pe-0"/>
+                        </div>
```

Comment:
Hello @kir-odoo, thanks a lot for the work on the product catalog 🙇 

It's odd to apply a `container p-0` and `col-12 pe-0` is there a reason to not use the `dropdown-item` design here?

Reply:
https://github.com/odoo/odoo/pull/198489#discussion_r2018722487
As explained here, it didn't work with the dropdown menu item; boolean_favorite is overriding its own CSS, so it isn't able to click.